### ISSUE TITLE
cos-cgroupv2-containerd-node-e2e-serial failing after feature gate was GA'd

### ISSUE THREAD

**Issue by bobbypage** [kubernetes/kubernetes#114475]
### What happened?

cos-cgroupv2-containerd-node-e2e-serial is failing. 

Example: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-cos-cgroupv2-containerd-node-e2e-serial/1602401700964995072

```
I1212 20:47:35.224] ERROR: initialize feature gates: unrecognized feature gate: LocalStorageCapacityIsolation
```

This is appears due to https://github.com/kubernetes/kubernetes/pull/114410 which removed the `LocalStorageCapacityIsolation` feature gate after it was GA'd

We should probably update the test-infra config to avoid setting this feature gate now since it is GA.

### What did you expect to happen?

n/a

### How can we reproduce it (as minimally and precisely as possible)?

n/a

### Anything else we need to know?

n/a

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### Cloud provider

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by akhilerm** [comment]
The following tests are also failing due to the missing feature gate flag:
- https://testgrid.k8s.io/sig-node-containerd#node-kubelet-containerd-hugepages
- https://testgrid.k8s.io/sig-node-containerd#node-kubelet-containerd-performance-test
- https://testgrid.k8s.io/sig-node-containerd#node-kubelet-containerd-resource-managers

**Comment by aojea** [comment]
yeah, same here https://github.com/kubernetes/kubernetes/issues/114449
